mobile/assets/i18n/en-US.json

```diff
@@ -506,6 +506,8 @@
   "theme_setting_theme_title": "Theme",
   "theme_setting_three_stage_loading_subtitle": "Three-stage loading might increase the loading performance but causes significantly higher network load",
   "theme_setting_three_stage_loading_title": "Enable three-stage loading",
+  "thumbnail_alt_text_date": "Image taken on {}",
+  "thumbnail_alt_text_date_and_location": "Image taken in {} on {}",
```

Comment:
Thanks for the feedback - I agree that it would be nice to have the people information in the alt text, just like web.

Person data currently isn't used in the asset grid, but it should be possible to pull from the remote assets.

Reply:
Hey @mertalev, I've done some looking into this. I don't think it's possible to add tagged people information to the alt text without a server change. The `/api/sync/full-sync` endpoint doesn't include person information, and this is the data that's stored in the mobile database to render out the timeline. 

What do you think about leaving the mobile alt text as-is for now?

Reply:
I'm totally fine with that! The person names are just a nice-to-have.